packages/editor/src/lib/editor/Editor.ts

```diff
@@ -5158,7 +5158,7 @@ export class Editor extends EventEmitter<TLEventMap> {
 		point: VecLike,
 		opts = {} as {
 			renderingOnly?: boolean
-			margin?: number
+			margin?: number | [number, number] // [inside, outside]
```

Comment:
mind pulling this out as its own interface and doing this comment as a proper doc comment?

Reply:
Or you can also write `[inside: number, outside: number]` which is a weird typescript syntax but useful documentation